definitions/output/core_web_vitals/technologies.js

```diff
@@ -30,8 +30,51 @@ CREATE TEMP FUNCTION IS_NON_ZERO(
 ) RETURNS BOOL AS (
   good + needs_improvement + poor > 0
 );
+
+CREATE TEMP FUNCTION extract_audits (lighthouse JSON)
```

Comment:
@rviscomi please give me feedback here, if we understand performance opportunities the same way.

Reply:
LMK if https://github.com/HTTPArchive/dataform/pull/38#issuecomment-2596450415 helps